packages/itwin/measure-tools/src/api/SheetMeasurementHelper.ts

```diff
@@ -0,0 +1,84 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+import { ColorDef, QueryBinder } from "@itwin/core-common";
+import type { DecorateContext, GraphicBuilder, IModelConnection } from "@itwin/core-frontend";
+import { GraphicType } from "@itwin/core-frontend";
+import type { Point3d } from "@itwin/core-geometry";
+import { Point2d } from "@itwin/core-geometry";
+
+interface IdAndLocation {
```

Comment:
This looks very similar to the `DrawingMetadata` interface. I'd consolidate to just using the one defined in Measurement now.

Reply:
(Not to mention, this should have been exported since I see it's used in an exported function below)